compiler-core/src/ast.rs

```diff
@@ -1286,7 +1286,7 @@ impl<T> HasLocation for CallArg<T> {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct RecordUpdateSpread {
+pub struct RecordUpdateRecord {
```

Comment:
search-replace accident? 😁

Reply:
No, no, I went through all occurrences manually and used the LSP for renaming. I thought this represents the _record_ part of the record update, e.g. in `let new_wibble = Wibble(..old_wibble, wobble_factor: 10)` this would be `..old_wibble`. Is that assumption correct? Do you have a better name?

Reply:
Sounds good, I'll fix that after work. Let me know if I missed anything else until then :)

Reply:
Thank you!